python/tk_multi_workfiles/actions/open_file_action.py

```diff
@@ -232,28 +258,25 @@ def _open_in_current_work_area(self, src_path, src_template, file, src_work_area
         # copy and open the file:
         return self._do_copy_and_open(src_path, 
                                       dst_file_path, 
-                                      version = None, 
-                                      read_only = False, 
-                                      new_ctx = dst_work_area.context, 
-                                      parent_ui = parent_ui)
+                                      version=fields.get("version"),
+                                      read_only=False,
+                                      new_ctx=dst_work_area.context,
+                                      parent_ui=parent_ui)
 
 class ContinueFromFileAction(OpenFileAction):
     """
     """
-    def __init__(self, label, file, file_versions, environment, next_version=None):
+    def __init__(self, label, file, file_versions, environment, next_version_override):
```

Comment:
please add a docstring. Is next version override a function, a string or an int?